client/views/events/edit/event.edit.app-test.js

```diff
@@ -0,0 +1,58 @@
+import { Meteor } from 'meteor/meteor';
+import { assert } from 'meteor/practicalmeteor:chai';
+
+if (Meteor.isClient) {
+	describe('Event save', function() {
+		let eventId;
+		let event;
+
+		it('Stores an event', function(done) {
+			Meteor.call('login', {
+				"user": {
+					"username": "greg"
+				},
+				"password": {
+					"digest": "0d2c690e7dd5f94780383e9dfa1f4def044319104ad16ab15e45eeb2a8dfc81b",
+					"algorithm": "sha-256"
+				}
+			});
```

Comment:
`Meteor.call` without callback. If something fails here, that failure might remain hidden.